mldsa/fips202/fips202.h

```diff
@@ -35,6 +35,14 @@ typedef struct
 extern const uint64_t KeccakF_RoundConstants[];
 
 #define shake128_init FIPS202_NAMESPACE(shake128_init)
+/*************************************************
+ * Name:        shake128_init
+ *
+ * Description: Initializes Keccak state for use as SHAKE128 XOF
```

Comment:
nit: The mentioning of `Keccak` feels misplaced here now that the context is generic.